Lib/test/test_dict.py

```diff
@@ -1452,6 +1452,25 @@ def items(self):
         d = CustomReversedDict(pairs)
         self.assertEqual(pairs[::-1], list(dict(d).items()))
 
+    @support.cpython_only
+    def test_dict_items_result_gc(self):
+        # bpo-42536: dict.items's tuple-reuse speed trick breaks the GC's
+        # assumptions about what can be untracked. Make sure we re-track result
+        # tuples whenever we reuse them.
+        i = iter({None: []}.items())
```

Comment:
nitpick: i suggest to rename the variable to "it", I dislike names of a single letter ;-) Same remark for other tests.